packages/react-core/src/components/Pagination/PaginationOptionsMenu.tsx

```diff
@@ -66,7 +66,6 @@ export class PaginationOptionsMenu extends React.Component<PaginationOptionsMenu
     firstIndex: 0,
     lastIndex: 0,
     defaultToFullPage: false,
-    itemCount: 0,
     itemsTitle: 'items',
     toggleTemplate: ({ firstIndex, lastIndex, itemCount, itemsTitle }: ToggleTemplateProps) => (
       <React.Fragment>
```

Comment:
I think that you may need to update Pagination.tsx so that the `paginationTotalItems` is also consuming the user passed `toggleTemplate`. That is why the 0 is still in the smaller viewports